app/assets/stylesheets/admin/dashboard/_dashboard.scss

```diff
@@ -0,0 +1,15 @@
+.admin-dashboard {
+  .content {
+    .row {
+      margin-bottom: 1em;
+    }
+
+    section {
+      section {
```

Comment:
This seems like an odd pattern and perhaps a bit brittle. Can you just use a class on the inner section perhaps?